alerting/src/main/kotlin/org/opensearch/alerting/AlertService.kt

```diff
@@ -181,15 +186,36 @@ class AlertService(
         findings: List<String>,
         relatedDocIds: List<String>,
         ctx: DocumentLevelTriggerExecutionContext,
-        alertError: AlertError?
+        alertError: AlertError?,
+        workflowExecutionId: String? = null,
```

Comment:
should we name this to execution id? Changing the name here makes it more confusing.